lib/printers/pretty.js

```diff
@@ -22,30 +22,47 @@ class PrettyPrinter {
     }
 
     let warningCount = this.options.quiet ? 0 : results.warningCount;
-    let count = results.errorCount + warningCount;
+    let todoCount = this.options.quiet || !this.options.includeTodo ? 0 : results.todoCount;
```

Comment:
So if we have includeTodo set to true and quiet, this is going to remove the todos from output. Do we want that? If you set includeTodo, I feel like you're explicitly asking to see the todos.